tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -3,6 +3,7 @@ import path from 'path';
 
 import yazl from 'yazl';
 import yauzl from 'yauzl';
+import * as tmp from 'tmp-promise';
```

Comment:
@marchiore 
Nit, I don't think that using `import * as tmp from '...'` here is necessary,
`import tmp from 'tmp-promise';` should work as well (in this case they should be equivalent).

doesn't `import tmp from 'tmp-promise` work for you as expected?

Reply:
Changed it!